R/pkg/vignettes/sparkr-vignettes.Rmd

```diff
@@ -906,6 +910,24 @@ predicted <- predict(model, df)
 head(predicted)
 ```
 
+#### FP-growth
+
+`spark.fpGrowth` executes FP-growth algorithm to mine frequent itemsets on a `SparkDataFrame`.
+
+* `spark.freqItemsets` method can be used to retrieve a `SparkDataFrame` with the frequent itemsets.
+* `spark.associationRules` returns a `SparkDataFrame` with the association rules.
+
+
+```{r}
+items <- selectExpr(createDataFrame(data.frame(items = c(
+  "s,t,u",
```

Comment:
What do you mean by "real"? Something human readable (e.g. milk, bread, butter) or some standard pattern mining dataset? If the former one then it is not a problem. If the latter one I am not aware of any dataset which would be safe enough on the license side.